akka-persistence/src/main/mima-filters/2.5.6.backwards.excludes

```diff
@@ -1,2 +1,5 @@
 # #23436 AbstractPersistentLoggingFSM, this is not binary compatible but previous was not possible to use at all 
 ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("akka.persistence.fsm.AbstractPersistentFSM.domainEventClass")
+
+# #18470 Calls to stash() inside of persist callbacks should throw exception
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.persistence.Eventsourced.akka$persistence$Eventsourced$$super$stash")
```

Comment:
hm is this still needed? seems we just added an override which should be fine hm

Reply:
Ping about this one, still needed?

Reply:
it is needed, mima report says
`synthetic method akka$persistence$Eventsourced$$super$stash()Unit in trait akka.persistence.Eventsourced is present only in current version`

Reply:
@ortigali sorry for review delays, this should now be in 2.5.7.backwards.excludes